Podfile

```diff
@@ -11,7 +11,7 @@ plugin 'cocoapods-art', :sources => [
   'tomtom-sdk-cocoapods'
 ]
 
-TOMTOM_SDK_VERSION = '0.2.3404'
+TOMTOM_SDK_VERSION = '0.13.0'
 
 target 'TomTom SDK Examples' do
```

Comment:
The best way is to eliminate spaces between directory and target names.
I don't think any iOS developer would intentionally do this, and these spaces will be a permanent item on any issue route cause analysis list.

Reply:
You are right 👍 Spaces are removed from the target name, project name, and directories.

Reply:
Thank you for bringing the issue to our attention! All problems related to space characters are resolved!